scripts/ct/run.sh

```diff
@@ -80,6 +80,17 @@ if [ "${WHICH_APP}" = 'novalue' ]; then
     exit 1
 fi
 
+beginswith() { case $2 in "$1"*) true;; *) false;; esac; }
+
+if beginswith lib-ee "${WHICH_APP}" && { [ -z "${PROFILE+x}" ] || [ "${PROFILE}" != emqx-enterprise ]; }
```

Comment:
```suggestion
if [[ "${WHICH_APP}" == lib-ee* && ( -z "${PROFILE+x}" || "${PROFILE}" != emqx-enterprise) ]]; then
```

then `beginswith` function is not needed